pyproject.toml

```diff
@@ -40,6 +40,7 @@ default_section = 'THIRDPARTY'
 force_grid_wrap = 0
 include_trailing_comma = true
 known_first_party = 'datadog_checks'
+src_paths = ''
 line_length = 120
 multi_line_output = 3
 skip_glob = 'datadog_checks/dev/tooling/signing.py,datadog_checks/dev/tooling/templates/*,datadog_checks/*/vendor/*'
```

Comment:
Looks like skip_glob is not working anymore with isort v5, see: https://github.com/PyCQA/isort/issues/1468